internal/api/settings.go

```diff
@@ -73,7 +75,9 @@ func (a *API) Settings(w http.ResponseWriter, r *http.Request) error {
 			Zoom:         config.External.Zoom.Enabled,
 		},
 		HookConfiguration: HookSettings{
-			MFAVerification: config.Hook.MFAVerificationAttempt.Enabled,
+			MFAVerificationAttempt:      config.Hook.MFAVerificationAttempt.Enabled,
+			PasswordVerificationAttempt: config.Hook.PasswordVerificationAttempt.Enabled,
+			CustomAccessToken:           config.Hook.CustomAccessToken.Enabled,
```

Comment:
Let's not make this information public. We need to think about this settings endpoint better -- it's not useful.